development/idp-local/config/config.php

```diff
@@ -46,7 +46,6 @@
 $ADMIN_NAME = Env::get('ADMIN_NAME', 'SAML Admin');
 $ADMIN_PROTECT_INDEX_PAGE = Env::get('ADMIN_PROTECT_INDEX_PAGE', true);
 $SHOW_SAML_ERRORS = Env::get('SHOW_SAML_ERRORS', false);
-$TIMEZONE = Env::get('TIMEZONE', 'GMT');
```

Comment:
Wouldn't it have been easier to hardcode this, rather than change all 'timezone' references below?

Reply:
Perhaps